janitor/functions.py

```diff
@@ -57,6 +57,7 @@ def clean_names(
     strip_underscores: str = None,
     case_type: str = "lower",
     remove_special: bool = False,
+    original_columns: bool = True,
```

Comment:
@shantanuo thanks for adding it in. However, when reading the API, with `original_columns`, it does not read as consistently as `preserve_original_columns`. (The more general idea is to provide an 'action term', which is what makes the API declarative.) 

The API really matters here, for that is pyjanitor's biggest contribution to the community. Could you change this to something that reads as an action term, e.g. `preserve_original_columns`, or `record_original_columns` or `backup_original_columns`? Do try to consider using the least ambiguous term across cultures, if possible.